manifests/cleanup-cni.yaml

```diff
@@ -0,0 +1,21 @@
+# This MachineConfig adds file /usr/lib/tmpfiles.d/cleanup-cni.conf
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 51-cleanup cni configuration files at boot
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          # r /etc/origin/openvswitch/conf.db
+          # r /etc/cni/net.d/80-openshift-network.conf
+          # r /etc/cni/net.d/10-ovn-kubernetes.conf
+          source: data:,r%20/etc/origin/openvswitch/conf.db%0Ar%20/etc/cni/net.d/80-openshift-network.conf%0Ar%20/etc/cni/net.d/10-ovn-kubernetes.conf
+        filesystem: root
+        mode: 0644
+        path: /usr/lib/tmpfiles.d/cleanup-cni.conf
```

Comment:
what is this doing? just deleting the paths above? modifying them? do you need this at every boot? every reboot? initial boot at installer time?

Reply:
Runs at every boot to delete leftover files from the last time the host was running